.vscode/settings.json

```diff
@@ -18,5 +18,5 @@
         "./bazel-out/_coverage/_coverage_report.dat"
     ],
     "python.formatting.provider": "yapf",
-    "editor.formatOnSave": true
```

Comment:
any reason to turn off this?

Reply:
If not opened at the root of the XLA repo where the yapf style resides, this can lead to unintended reformatting. I saw at least one commit on this branch that was an unintended reformat of the whole script. 
It's unrelated to the rest, so I can remove this if you prefer it enabled.

Reply:
It's just that you may get push back when merging the benchmark branch to the master branch.

fwiw, on master branch, the CI checks the linter via something like `yapf -i -r *.py test/ scripts/ torch_xla/` and we haven't seen `editor.formatOnSave` causing an issue so far.

Reply:
Whenever you are ready, feel free to create a pr to merge this branch to the master branch.

Reply:
Thanks for the heads up!